DNN Platform/Connectors/GoogleTagManager/Dnn.GoogleTagManagerConnector.sln

```diff
@@ -0,0 +1,31 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.30503.244
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Dnn.GoogleTagManagerConnector", "Dnn.GoogleTagManagerConnector.csproj", "{ADB57FFD-677C-4C30-9723-6EE0A475315B}"
+EndProject
+Global
+	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Cloud_Debug|Any CPU = Cloud_Debug|Any CPU
+		Cloud_Release|Any CPU = Cloud_Release|Any CPU
+		Debug|Any CPU = Debug|Any CPU
+		Release|Any CPU = Release|Any CPU
+	EndGlobalSection
```

Comment:
The module itself should only have a project file and be included in the main solution file, can this solution file be removed.

Reply:
Sure.